content/threads_intro.tex

```diff
@@ -0,0 +1,74 @@
+This section specifies the interaction between \openshmem{} interfaces and the
+user threads, and also describes the routines that can be used for initializing and 
+querying the thread environment. There are four levels of threading supported by
+the \openshmem{} implementation.
+ 
+\begin{itemize}
+\item {\bf \CONST{SHMEM\_THREAD\_SINGLE}} The \openshmem{} program may not be multithreaded, 
+and all \openshmem{} interfaces are invoked by a single thread. 
+
+\item {\bf \CONST{SHMEM\_THREAD\_FUNNELED}}
+The \openshmem{} program may be multithreaded, however, the 
+program must ensure that only the main thread invokes the \openshmem{}
+interfaces.
+
+\item {\bf \CONST{SHMEM\_THREAD\_SERIALIZED}} 
+The \openshmem{} program may be multithreaded, however, the 
+program must ensure that only one thread invokes the \openshmem{}
+interfaces at any given instance i.e., the \openshmem{} interfaces 
+are not invoked concurrently by multiple threads.
+
+\item {\bf \CONST{SHMEM\_THREAD\_MULTIPLE}}
+The \openshmem{} program may be multithreaded, and any 
+thread may invoke the \openshmem{} interfaces.
+\end{itemize}
+
+\hspace{-15pt}
+{\bf Semantics}
+
+\begin{itemize}[leftmargin=*]
+
+\item
+In the \openshmem{} program, a thread calling \FUNC{SHMEM\_INIT} is designated as the main
```

Comment:
Sure, I will fix it. Are we following some convention ? Upper case in the function descriptions and lower case in back matter / front matter ? From one of the previous comment, I thought we had a convention of using lower case in function descriptions.